src/components/v5/shared/Button/TextButton.tsx

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable react/button-has-type */
+import React, { FC, PropsWithChildren } from 'react';
+import { useIntl } from 'react-intl';
+import clsx from 'clsx';
+
+import { TextButtonProps } from './types';
+import SpinnerLoader from '~shared/Preloaders/SpinnerLoader';
+import styles from './TextButton.module.css';
+
+const displayName = 'v5.Button.TextButton';
```

Comment:
v5.shared.Button.TextButton

Reply:
As I can see, we have never used the prefix `shared` for components inside this directory.